### PR TITLE
Issue #49: Upload complete page

### PR DESCRIPTION
**Pull request by JoshuaBuditama**
This pull request addresses issue #49 .
When a verification upload process is successful, a pop-up message that indicates the success will be displayed.
Currently it's displayed on top of the mock device display. Unsure of the cause of this.
If it's an issue, I will be more than happy to try to fix the display so that the success message is displayed inside the container.

And also, I put the tests in the same folder as the components.
We might need to move all of the tests in a separate folder eventually to make the files to be more organised but I believe that this should be addressed in a separate issue.

### CODE REVIEW

**Comment by JakeBrown** [comment]
@JoshuaBuditama I think the problem is trying to use Bulma's built in modal class for this. It's going to show the modal over the whole screen rather than within a parent element. 

You could probably hack this with CSS to override Bulma but better options might be:

- Don't use a modal, just use React to switch out the content of your page. 
- Use JavaScript to show a popup. Eg. `alert()` or `confirm()`. This is implemented at the browser level so the result will look different on different browsers. Not a total fix but less code for the same result.

**Comment by JoshuaBuditama** [comment]
@JakeBrown Thanks heaps for your suggestions!
What I ended up doing was just replacing the whole page with the success message on successful submission haha.
It works sort of like moving to another page except that it actually does not.
Would this be fine?

Edit: I just realised that what I did was exactly what you suggested in your first point. So hopefully this fix is ok haha.

File: front-end/src/components/UploadForm.tsx [hunk 1]
```diff
@@ -17,73 +18,85 @@ function UploadForm(){
...
-            <div className="columns is-centered">
-              {/*Cancel button*/}
-              <div className="column is-half">
-                <div className="field">
-                  <p className="control">
-                    <button className="button">
-                      Cancel
-                    </button>
-                  </p>
+      {/**Notification that appears when the token submission is successful*/}
```
Comment (JakeBrown) [inline front-end/src/components/UploadForm.tsx:35]: It looks like there are two different pages with no shared content? In that case, you can just have a conditional return statement which makes this really clear:

```
if (submitSuccess) {
return (
 <div className="column is-half"> <UploadSuccess onPopUpClick={handlePopUpClick}/>...
)
} 

else {
return (
<section className="section is-medium has-text-centered">...
}

```

Your solution achieves the same thing, this is just a little clearer.

**Review by JakeBrown — APPROVED** [review]
Looks good! One suggestion to improve code readability, but definitely not mandatory.

File: front-end/src/components/UploadForm.tsx (hunk 1 above)
Comment (JoshuaBuditama) [reply front-end/src/components/UploadForm.tsx:35]: Thanks for the tip Jake!
I will keep that in mind 👍

**Comment by JoshuaBuditama** [comment]
Closes #49

**Pull request merged**